src/Symfony/Component/DependencyInjection/Compiler/RepeatedPass.php

```diff
@@ -56,13 +56,14 @@ public function __construct(array $passes)
      */
     public function process(ContainerBuilder $container)
     {
+repeat:
```

Comment:
fabbot complains about the indentation here. Can it be indentation or will it break ?